### PR TITLE
Diversity acronyms

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

File: etc/glossarydefs.csv [hunk 1]
```diff
@@ -569,6 +580,7 @@ PMM,precision measuring machine,TS,Document-11988,,A
 PMO,\gls{Project Management Office},Adm,,,A
 PMP,(DM) Project Management Plan; LDM-294,LSST DM,,,A
 POC,Proof Of Concept,Gen,,,A
+POCIT,People Of Color In Tech,DEI,,,A
```
Comment (ranpalgill) [inline etc/glossarydefs.csv:583]: POC People of Color to be added

File: etc/glossarydefs.csv [hunk 2]
```diff
@@ -583,6 +595,7 @@ PSF,Point Spread Function,Gen,,,A
 PST,\gls{Project Science Team},LSST DM,,,A
 PSTN,Project Science Technical Note,LSST DM,,,A
 PVI,Processed Visit Image,DM,,,A
+PWI,Predominantly White institution,DEI,,,A
```
Comment (ranpalgill) [inline etc/glossarydefs.csv:598]: Capitalise Institution

File: etc/glossarydefs.csv [hunk 3]
```diff
@@ -768,9 +784,11 @@ WAN,Wide Area Network,Gen,,,A
 WBS,\gls{Work Breakdown Structure},Gen,,,A
 WCAG,Web Content Accessibility Guidelines,OPS,,,A
 WCS,\gls{World Coordinate System},Gen,,,A
+WEPAN,Women in Engineering ProAvtive Network,DEI,,,A
```
Comment (ranpalgill) [inline etc/glossarydefs.csv:787]: typo ProAvtive should be ProActive

File: etc/glossarydefs.csv (hunk 1 above)
Comment (womullan) [reply etc/glossarydefs.csv:583]: ok

File: etc/glossarydefs.csv (hunk 2 above)
Comment (womullan) [reply etc/glossarydefs.csv:598]: ok

**Pull request merged**